src/backend/test/support/factories.js

```diff
@@ -0,0 +1,19 @@
+const faker = require("faker");
+const { Factory } = require("rosie");
+const { sample } = require("lodash");
+const Firefighter = require("../../models/firefighter");
+
+const firefightersFactory = new Factory().attrs({
+  name: faker.name.firstName,
+  email: faker.internet.email(),
+  password: "foobar",
+  status: sample("active", "inactive", "busy")
```

Comment:
We are missing one attr that is `dutyType`

Reply:
Not need for the tests though, wanna add it?

Reply:
Besides, dont write reviews here, this is a long standing PR :stuck_out_tongue:

Reply:
I know, but had nowhere else to comment. And duty is not working in the app :/